compiler/rustc_mir_transform/src/coverage/graph.rs

```diff
@@ -316,11 +316,6 @@ impl BasicCoverageBlockData {
     pub fn last_bb(&self) -> BasicBlock {
         *self.basic_blocks.last().unwrap()
     }
-
-    #[inline(always)]
-    pub fn terminator<'a, 'tcx>(&self, mir_body: &'a mir::Body<'tcx>) -> &'a Terminator<'tcx> {
-        &mir_body[self.last_bb()].terminator()
-    }
```

Comment:
With the change to passing around BBs, this method ended up being used only from one test, so it was less hassle to just inline and delete it.